test/service_objects/game_piece_action_resolvers/ranged_attack_resolver.test.ts

```diff
@@ -184,35 +198,37 @@ describe('resolveRangedAttackAction', () => {
 
       // Check action to now be resolved with saved results
       await action.reload();
-      expect(action.actionData['resolved']).toBe(true)
-      let savedResolvedAttacks = action.actionData['resolvedAttacks']
-      expect(savedResolvedAttacks.length).toBe(1)
-      expect(savedResolvedAttacks[0].hitRoll.roll).toBe(6)
-      expect(savedResolvedAttacks[0].hitRoll.success).toBe(true)
-      expect(savedResolvedAttacks[0].woundRoll.roll).toBe(6)
-      expect(savedResolvedAttacks[0].woundRoll.success).toBe(true)
-      expect(savedResolvedAttacks[0].saveRoll.roll).toBe(1)
-      expect(savedResolvedAttacks[0].saveRoll.success).toBe(false)
-      expect(savedResolvedAttacks[0].damageDealt).toBe(1)
-
-      expect(action.actionData['totalDamageDealt']).toBe(1);
+      expect(action.actionData['resolved']).toBe(true);
+      let savedResolvedAttacks = action.actionData['resolvedAttacks'];
+      expect(savedResolvedAttacks.length).toBe(3);
```

Comment:
@louiswheeleriv the unit in this test has 3 attacks, and currently, it doesn't work if a unit submits fewer than their total number of attacks.

Reply:
Sounds good. I don't think I ever actually added validation for that but it should always be the case that a unit makes that number of attacks determined by the associated stat.